renderer/functions.go

```diff
@@ -168,10 +169,14 @@ func (k *kubernetesHelper) LinkForKubeType(t *types.Type) string {
 	if len(parts) < 2 {
 		zap.S().Fatalw("Unexpected Kubernetes package name", "type", t)
 	}
-
+	// TODO: if there're more alaises, we should export them to the config
```

Comment:
I'd like to avoid to add TODO in the code. Where could we look to see if there are other aliases?
```suggestion
	// TODO: if there're more aliases, we should export them to the config
```
Otherwise, maybe just add a comment above the "if group == "apiextensions"" indicating that this is alias handling;